k8s/namespaces/rd/rd-caseworker-ref-api-integration/rd-caseworker-ref-api-integration.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/rd/caseworker-ref-api:prod-ea5bf8f-20210824110521
+      image: hmctspublic.azurecr.io/rd/caseworker-ref-api:prod-ea5bf8f-20210824110521   #{"$imagepolicy": "flux-system:rd-caseworker-ref-api-integration"}
```

Comment:
Can you change this policy to `rd-caseworker-ref-api`? It looks to be using the same, I would remove the image repo/policy for `rd-caseworker-ref-api-integration`

Reply:
Yes the integration applications are using the same image.  Image policy/repo now updated.